src/logger/mongoose_record_pp.erl

```diff
@@ -0,0 +1,19 @@
+-module(mongoose_record_pp).
```

Comment:
Maybe some documentation about what this new module is for?

Reply:
done